src/libcore/macros.rs

```diff
@@ -337,21 +337,16 @@ macro_rules! try {
 
 /// Write formatted data into a buffer
 ///
-/// This macro accepts a 'writer' (any value with a `write_fmt` method), a format string, and a
-/// list of arguments to format.
+/// This macro accepts a format string, a list of arguments, and a 'writer'.
+/// Arguments will be formatted according to the specified format string and the result will be
```

Comment:
I actually didn't (I just return after sentences out of habbit so that changes to one sentence in the future don't necessarily cause the whole paragraph to be rewrapped and make the diff longer); happy to add a paragraph if you think it should be two, but it felt like a single coherent description to me (and the docs online currently are hard to read because it's mostly "single line" paragraphs, meaning there's as much whitespace in the text as text itself). Personally, I thought it was easier to read this way, but as I said, if you're sure it should be chagned I'm happy to split it out into single sentence paragraphs.